### PR TITLE
Add option to sleep after meditation.

### PR DESCRIPTION
**Pull request by ghub12v**

### CODE REVIEW

**Comment by ghub12v** [comment]
Please review this, @casey-collab

File: english/meditation/meditation.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 You should relax, you completed the git course, good job!
+You feel relaxed and sleepy... Go to sleep?
 
-Click [here](https://www.youtube.com/watch?v=2u86NS3NtMw)!
\ No newline at end of file
+Click [here](https://www.youtube.com/watch?v=2u86NS3NtMw)!
+[Go to sleep](expired-milk/expired-milk.md)?
```
Comment (casey-collab) [inline english/meditation/meditation.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline english/meditation/meditation.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ghub12v** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ferris_wheel:

**Pull request merged**